src/api/v3/events.py

```diff
@@ -0,0 +1,91 @@
+from django.http import JsonResponse
+
+from backend.settings import WEBAPP_URL, TIME_ZONE
+from cms.models import Region
+from .locations import transform_poi
+from ..decorators import json_response
+
+
+def transform_event(event):
+    """
+    Function to create a JSON from a single event object.
+
+    :param event: The event which should be converted
+    :type event: ~cms.models.events.event.Event
+
+    :return: return data necessary for API
+    :rtype: dict
+    """
+    return {
+        "id": event.id,
+        "start_date": event.start_date,
+        "end_date": event.end_date,
+        "all_day": event.is_all_day,
+        "start_time": event.start_time,
+        "end_time": event.end_time,
+        "recurrence_id": event.recurrence_rule.id if event.recurrence_rule else None,
+        "timezone": TIME_ZONE,
+    }
+
+
+def transform_event_translation(event_translation):
+    """
+    Function to create a JSON from a single event_translation object.
+
+    :param event_translation: The event translation object which should be converted
+    :type event_translation: ~cms.models.events.event_translation.EventTranslation
+
+    :return: return data necessary for API
+    :rtype: dict
+    """
+
+    event = event_translation.event
+    if event.location:
+        location = transform_poi(event.location, event_translation.language.slug)
```

Comment:
```suggestion
        location_translation = (
            event.location.get_public_translation(event_translation.language.slug)
            or event.location.default_translation
        )
        location = transform_poi(event.location, location_translation)
```
Keep in mind that we don't know if there is a translation of the location in the same language as the event translation. Thus, we have to add a fallback to the default translation of the POI, otherwise you might get an `AttributeError: 'NoneType' object has no attribute 'title'`.